### ISSUE TITLE
Make message expiration time configurable

### ISSUE THREAD

**Issue by dschwen** [nuvie/nuvie#37]
In the _new_ UI mode message that appear in the top left box (when issuing commands like Look or Use) disappear way too fast. The timeout for them should be a config option (maybe the last messages should be made sticky).

**Comment by yuv422** [comment]
Good point. I'll look at adding some more config options around the message box. The auto scrolling is a bit jarring at times.

It is also visible when you talk to an NPC which doesn't look nice when it retracts behind the converse gump.

**Comment by dschwen** [comment]
Which class handles that message box? I was a bit lost in the code. It is not the Console object, right?

**Comment by dschwen** [comment]
Looks like it is the ```MsgScroll``` or ```MsgScrollNewUI``` class.

**Comment by yuv422** [comment]
Yes the MsgScrollNewUI class handles the message box. It extends the MsgScroll class. MsgScroll formatting is quite fragile unfortunately. :(
It needs to handle the explicit and implicit page breaks from the original game. Along with some other formatting.
